frontend/public/components/build-logs.jsx

```diff
@@ -1,8 +1,26 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
-import { ResourceLog, LOG_SOURCE_RUNNING, LOG_SOURCE_TERMINATED, LOG_SOURCE_WAITING } from './utils';
+import { LoadingInline, LOG_SOURCE_RUNNING, LOG_SOURCE_TERMINATED, LOG_SOURCE_WAITING, MsgBox, ResourceLog } from './utils';
+import { getJenkinsLogURL } from './build-pipeline';
 
-const buildToLogSourceStatus = (phase) => {
+const PipelineLogMessage = props => {
+  const logURL = getJenkinsLogURL(props.obj);
+  const message = logURL
+    ? 'Pipeline build logs are available through Jenkins (linked below)'
+    : 'A link to the Jenkins pipeline build logs will appear below when they are available (this may take some time)';
+  const link = logURL
+    ? <a href={logURL} target="_blank" rel="noopener noreferrer">View Log</a>
+    : <LoadingInline/>;
```

Comment:
I still don't like the loading indicator here. It makes it seem like the page is still loading and something is about to appear, which might not be the case.